runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerRecord.java

```diff
@@ -44,13 +44,23 @@
 	 * @param id other record id
 	 * @return true if match is made
 	 */
-	boolean matches(final String id, LootRecordType type)
+	boolean matches(final String id, LootRecordType type, String searchFilter)
 	{
+		if (!title.toLowerCase().contains(searchFilter.toLowerCase().trim()))
+		{
+			return false;
+		}
+
 		if (id == null)
 		{
 			return true;
 		}
 
 		return title.equals(id) && this.type == type;
 	}
+
+	boolean matches(final String id, LootRecordType type)
```

Comment:
I'd put this on L41 so the overload with more arguments appears later in the file.

Reply:
👍